### DISCUSSION TITLE
Feature request: add an option in settings to specify root directory

### DISCUSSION THREAD

**Discussion by AlexIII** [yandeu/five-server#43] [General]
It would be useful to have the ability to specify root directory which will be served when clicking "Go Live" button. 

settings.json:
```
"fiveServer.root": "./web"
```

**Comment by yandeu** [comment]
This is possible. Please read https://github.com/yandeu/five-server#config-file.

**Reply by AlexIII** [reply]
Thanks! I did not realize there's a separate config file as usually extensions work with `settings.json`. Will give it a try.
